test/Identity.Test/SignInManagerTest.cs

```diff
@@ -121,17 +123,16 @@ public async Task PasswordSignInReturnsLockedOutWhenLockedOut()
             var options = new Mock<IOptions<IdentityOptions>>();
             options.Setup(a => a.Value).Returns(identityOptions);
             var claimsFactory = new UserClaimsPrincipalFactory<PocoUser, PocoRole>(manager.Object, roleManager.Object, options.Object);
-            var logStore = new StringBuilder();
-            var logger = MockHelpers.MockILogger<SignInManager<PocoUser>>(logStore);
```

Comment:
@pakrym looks like something changed here. Regardless using TestLogger works reliably

Reply:
To add a bit more detail, the callback event on the mocked logger was no longer being invoked

Reply:
https://github.com/aspnet/KestrelHttpServer/pull/3066#issuecomment-434433211